test/Nancy.Tests/Unit/ModelBinding/DefaultBodyDeserializers/JsonBodyDeserializerFixture.cs

```diff
@@ -48,7 +48,7 @@ public void Should_report_false_for_can_deserialize_for_non_json_format()
             const string contentType = "application/xml";
 
             // When
-            var result = this.deserialize.CanDeserialize(contentType, A<BindingContext>._);
+            var result = this.deserialize.CanDeserialize(contentType, A.Dummy<BindingContext>());

```

Comment:
```C#
A<BindingContext>._
```
is designed for assertion only (e.g. `A.CallTo()`) which does not execute the `_` property.

In FakeItEasy 2.0, if `_` is ever executed, an exception is thrown. [Dummies](http://fakeiteasy.readthedocs.io/en/2.0.0/dummies/) are the simplest constructs for this use case.